### PR TITLE
[13.0][FIX] download invoice

### PR DESCRIPTION
**Pull request by david-popa**

### CODE REVIEW

**Comment by sbidoul** [comment]
Thanks! 

Could you also change the process method arguments ?

```
return self._process_method(
            service, "download", _id=_id, params=params
        )
```

should be

```
return self._process_method(
            service, "download", _id, params=params
        )
```

Regarding the auth I'm not convinced this is the correct solution. Actually do we know why this did work before (at least in v10) ?

File: shopinvader/controllers/main.py [hunk 1]
```diff
@@ -20,11 +20,11 @@ class InvaderController(main.RestController):
     _collection_name = "shopinvader.backend"
     _default_auth = "api_key"
 
-    @route(["/shopinvader/<service>/<int:_id>/download"], methods=["GET"])
+    @route(["/shopinvader/<service>/<int:_id>/download"], methods=["GET"], auth=_default_auth)
```
Comment (rousseldenis) [inline shopinvader/controllers/main.py]: Yes, I'm wondering why adding this as main.RestController is aware of the _default_auth...

**Review by lmignon — APPROVED** [review]

File: shopinvader/controllers/main.py (hunk 1 above)
Comment (lmignon) [reply shopinvader/controllers/main.py]: bc you in this case, you declare your own route not managed by the RestController

**Review by simahawk — APPROVED** [review]

**Comment by sbidoul** [comment]
Is there anything to do to not forget back and forward ports ? @Cedric-Pigeon

**Pull request merged**